en/micro-integrator/docs/overview/about-this-release-7.1.0.md

```diff
@@ -0,0 +1,98 @@
+# About this Release
+
+WSO2 Enterprise Integrator (EI) version 7.1.0 is the latest release of the WSO2 EI 7.x family, which provides a powerful solution for integrating systems.
+
+## What's new in this release?
```

Comment:
Can we mention something that caters to 6.x.x family users who might want to move over? I guess this release is meant to encourage those users to migrate so it might be good to say something that encourages that.